README.md

```diff
@@ -1,6 +1,53 @@
 # BMX
 
-BMX grants you API access to your AWS accounts, based on Okta credentials that you already own.
-It uses your Okta identity to create short-term AWS STS tokens, as an alternative to long-term IAM access keys.
+BMX provides API access to your AWS accounts using existing Okta credentials. Instead of long-term IAM access keys, BMX creates short-term AWS STS tokens using your Okta identity.
 
-This is a work in progress dotnet 7 branch
+# Installation
+
+Download the appropriate binary from the [releases](https://github.com/Brightspace/bmx/releases) page.
```

Comment:
Direct D2Lers to the internal site for install, same energy as: https://github.com/firebase/firebase-ios-sdk/blob/cc3ed01f70d479642f9d9d657c90c6677a030a3d/CONTRIBUTING.md?plain=1#L122-L123 etc...